### ISSUE TITLE
Package.json main field points to `index.js` instead of `index.cjs`

### ISSUE THREAD

**Issue by cachgill** [lezer-parser/javascript#2]
I ran into this issue when trying to get some tests to run with jest:

```
Cannot find module 'lezer-javascript' from 'index.js'
at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:265:17)
      at Object.<anonymous> (../../node_modules/@codemirror/next/lang-javascript/dist/index.js:10:24)
```

All of the other packages point to index.cjs in their main, and when built, dist does not contain an index.js in lezer-javascript.

**Comment by marijnh** [comment]
Right, that was an oversight. I didn't catch it because with node 13.5 and up it'll use the exports field instead. Released 0.8.4 with a fix.

**Comment by cachgill** [comment]
Thanks for the quick response.
